INSTALL.rst.txt

```diff
@@ -81,12 +78,8 @@ recent gfortran from an OS X package manager (like Homebrew).
 Please do NOT use gfortran from `hpc.sourceforge.net <http://hpc.sourceforge.net>`_,
```

Comment:
Line 76: Not a fan of recommending that dependencies are obtained from package managers. Links to gfortran binaries can be found at https://gcc.gnu.org/wiki/GFortranBinaries.